.gitmodules

```diff
@@ -79,3 +82,15 @@
 [submodule "deps/pareto"]
 	path = deps/pareto
 	url = https://github.com/chainblocks/pareto.git
+[submodule "deps/bgfx.cmake"]
+	path = deps/bgfx.cmake
+	url = https://github.com/chainblocks/bgfx.cmake.git
+[submodule "deps/brotli"]
```

Comment:
I kinda like externalproject, those used to be modules actually :D

Reply:
I like being able to use the target as part of the build system, it makes it easier to include another library (include paths & libs) without having to directly deal with library names and include paths.
It also easily allows you to switch any library between shared/static without having to change the lib names linked against.
Just thinking ahead a bit 😄

Reply:
Also iterating on changes in dependencies becomes easier as cmake checks for changes in them

Reply:
Yeah you are right, not sure though why I moved to externalproject.
But indeed if the external project uses cmake probably it's overkill!